<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cereon::RomUnitEditor</class>
 <widget class="QWidget" name="cereon::RomUnitEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="1">
    <widget class="QLineEdit" name="_contentLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="_addressLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="_sizeUnitComboBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="_sizeLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_sizeLabel">
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="_addressLabel">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="_browsePushButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="_contentLabel">
     <property name="text">
      <string>Content:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_addressLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>cereon::RomUnitEditor</receiver>
   <slot>_addressLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_sizeLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>cereon::RomUnitEditor</receiver>
   <slot>_sizeLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_sizeUnitComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>cereon::RomUnitEditor</receiver>
   <slot>_sizeUnitComboBoxCurrentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_browsePushButton</sender>
   <signal>clicked()</signal>
   <receiver>cereon::RomUnitEditor</receiver>
   <slot>_browsePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_addressLineEditTextChanged(QString)</slot>
  <slot>_sizeLineEditTextChanged(QString)</slot>
  <slot>_sizeUnitComboBoxCurrentIndexChanged(int)</slot>
  <slot>_browsePushButtonClicked()</slot>
 </slots>
</ui>
